spec/openapi.yaml

```diff
@@ -222,37 +119,62 @@ paths:
                 type: object
                 properties:
                   temperature:
-                    $ref: "#/components/schemas/OctoTemperatureState"
+                    $ref: "#/components/schemas/TemperatureState"
                   sd:
-                    $ref: "#/components/schemas/OctoSDState"
+                    $ref: "#/components/schemas/SDState"
                   state:
-                    $ref: "#/components/schemas/OctoPrinterState"
+                    $ref: "#/components/schemas/PrinterState"
+                  telemetry:
+                    oneOf:
+                      - $ref: "#/components/schemas/TelemetryMini"
+                      - $ref: "#/components/schemas/TelemetrySL1"
         403:
           $ref: "#/components/responses/Unauthorized"
+
   /api/printer/sd:
     get:
-      tags:
-        - octoprint
       summary: Retrieves the current state of the printer’s SD card.
-      security:
-        - ApiKeyAuth: []
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                type: object
-                $ref: "#/components/schemas/OctoSDState"
+                $ref: "#/components/schemas/SDState"
+        403:
+          $ref: "#/components/responses/Unauthorized"
+
+  # MINI
+  # - will probably not support this endpiont.
```

Comment:
typo error

Reply:
How to reproduce? Swagger editor didn't throw errors to me.

Reply:
Just a typo
endpiont -> endpoint

Reply:
fixed